commit 5389bada75b3cf386d5b6220b3a4185cb66c2c7c (refs/changes/61/1461/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-02-08T16:31:55+00:00 (2 years, 8 months ago)
    
    TOOLS-1684 sdcadm doesn't properly report an error if creating its logger fails

diff --git a/lib/cli/index.js b/lib/cli/index.js
index 1668cc5..2fa2f1d 100644
--- a/lib/cli/index.js
+++ b/lib/cli/index.js
@@ -121,9 +121,10 @@ CLI.prototype.init = function init(opts, args, callback) {
         });
     } catch (e) {
         if (e.code && e.code === 'ENOSPC') {
-            return callback('Not enought space to create log file');
+            return callback(new Error('Not enought space to create log file'));
         } else if (e.code && e.code === 'EACCES') {
-            return callback('Insufficient permissions to create log file');
+            return callback(new Error(
+                'Insufficient permissions to create log file'));
         } else {
             return callback(e);
         }
